package.json

```diff
@@ -38,7 +38,7 @@
     "test:unit": "jest",
     "test:playwright": "playwright test",
     "test:cypress": "start-server-and-test test:server 3000 test:do-cypress",
-    "test:do-cypress": "cypress run --project tests",
+    "test:do-cypress": "ELECTRON_EXTRA_LAUNCH_ARGS=--remote-debugging-port=8192 cypress run --project tests",
```

Comment:
We need to pass the `ELECTRON_EXTRA_LAUNCH_ARGS` stuff so that Cypress will know what port the devtools protocol is on. Electron for some reason doesn't pass this info along (and Electron is the default browser for Cypress runs). I'll make sure this gets added to our docs as well for users.